packages/example/src/pages/guides/configuration.mdx

```diff
@@ -143,6 +143,8 @@ To do so, provide your own [resolvers object](https://www.gatsbyjs.org/packages/
 
 To add a link to the bottom of each page that points to the current page source in GitHub, provide a `repository` object to `siteMetadata` in your `gatsby-config.js` file. You can provide a `baseUrl`, and if needed, the `subDirectory` and `branch` where your site source lives.
 
+To remove the link, remove the section that begins `options` through to its' closing brackets. If after deleting, your options object is empty, delete that as well.
```

Comment:
```suggestion
To disable the `Edit in Github` link, remove the `repository` option. If the options object is now empty, delete it all together.
```

Reply:
I'm happy with this - it's just that to me an "option" doesn't mean anything - I wouldn't know that that means I need to remove several lines of code 🤷‍♀️.
I'm probably not your usual audience though 🙂.